### PR TITLE
Fix encoding efficiency and document cleanup

### PR DESCRIPTION
**Pull request by Intrinsical-AI**
## Summary
- avoid gradient computation when encoding batches
- strip newline characters before indexing documents in Elastic

## Testing
- `python -m py_compile $(git ls-files '*.py')`

------
https://chatgpt.com/codex/tasks/task_e_685360b40f38832eb080a8a0a834641a

### CODE REVIEW

File: from_text_to_vectors/batch-sentence-transformers.py [hunk 1]
```diff
@@ -41,7 +41,8 @@ def batch_encode_to_vectors(input_filename, output_filename):
 
 
 def encode(documents):
-    embeddings = model.encode(documents, show_progress_bar=True)
+    with torch.no_grad():
```
Comment (Intrinsical-AI) [inline from_text_to_vectors/batch-sentence-transformers.py:44]: In this case the script it's not planning to train - so his should save memory avoid graph building and initial configuration

File: indexing_phase/indexer_elastic.py [hunk 2]
```diff
@@ -23,6 +23,7 @@ def index_documents(documents_filename, embedding_filename_first_field, embeddin
             # For each document creates a JSON document including both text and related vector.
             for index, (document, vector_string_384, vector_string_768) in enumerate(
                     zip(documents_file, vectors_file_384, vectors_file_768)):
+                document = document.strip()
```
Comment (Intrinsical-AI) [inline indexing_phase/indexer_elastic.py:26]: Avoid whitespaces / annoying new lines / noise

File: indexing_phase/indexer_elastic_with_pipeline.py [hunk 3]
```diff
@@ -17,6 +17,7 @@ def index_documents(documents_filename, client):
             documents = []
             # For each document creates a JSON document including text (and id).
             for index, document in enumerate(documents_file):
+                document = document.strip()
```
Comment (Intrinsical-AI) [inline indexing_phase/indexer_elastic_with_pipeline.py:20]: Avoid whitespaces / annoying new lines / noise
